en/integrations/plugin-packs/procedures/operatingsystems-linux-nrpe3.md

```diff
@@ -0,0 +1,275 @@
+| Memory                     |         X         |
+| Packet-Errors-Generic-Name |                   |
+| Packet-Errors-Global       |                   |
+| Process-Generic            |                   |
+| Swap                       |         X         |
+| Traffic-Generic-Name       |                   |
+| Traffic-Global             |                   |
+| Uptime                     |         X         |
+
+### Monitored metrics
+
+Here are the metrics monitored in the "Link by default" services.
+
+<!--DOCUSAURUS_CODE_TABS-->
+<!--Cpu-->
+
+| Metric          | UOM | Description               |
+| :-------------- | :-: | :------------------------ |
+| `cpu0`          |  %  | Average core 0 CPU usage  |
+| `cpu1`          |  %  | Average core 1 CPU usage  |
+| ...             |  %  | ...                       |
+| `total_cpu_avg` |  %  | Average overall CPU usage |
+
+<!--Load-->
+
+| Metric   | UOM | Description                            |
+| :------- | :-: | :------------------------------------- |
+| `load1`  |     | System load average on last 1 minute   |
+| `load5`  |     | System load average on last 5 minutes  |
+| `load15` |     | System load average on last 15 minutes |
+
+<!--Memory-->
+
+| Metric   | UOM | Description                           |
+| :------- | :-: | :------------------------------------ |
+| `buffer` |  B  | Amount of memory allocated to buffers |
+| `cached` |  B  | Amount of memory allocated to cache   |
+| `slab`   |  B  | Amount of memory allocated to Slab    |
+| `used`   |  B  | Total amount of used memory           |
+
+<!--Swap-->
+
+| Metric      | UOM | Description      |
+| :---------- | :-: | :--------------- |
+| `free`      |  B  | Free Swap space  |
+| `used`      |  B  | Used Swap space  |
+| `used_prct` |  %  | Swap space usage |
+
+<!--Uptime-->
+
+| Metric   | UOM | Description                    |
+| :------- | :-: | :----------------------------- |
+| `uptime` |  s  | Elapsed time since last reboot |
+
+<!--END_DOCUSAURUS_CODE_TABS-->
+
+## Prerequisites
+
+This chapter describes the installation prerequisites required by the plugin to run.
+
+### Network prerequisites
+
+The default TCP port for NRPE is 5666.
+
+| Source | Destination    | Protocol | Port |
+| ------ | -------------- | -------- | ---- |
+| Poller | Monitored host | TCP      | 5666 |
+
+### Plugin pack
+
+The plugin pack installation only impacts the central server and the procedure depends on the type of license.
+
+<!--DOCUSAURUS_CODE_TABS-->
+
+<!--IMP/EPP "Online" License & IT-100 Editions-->
+
+ Install the "Linux NRPE3" Plugin Pack from "Configuration > Plugin Packs > Manager" in the WUI.
+
+<!--IMP/EPP "Offline" License-->
+
+1. Install the Plugin Pack's RPM on the central server.
+
+```bash
+yum install centreon-pack-operatingsystems-linux-nrpe3
+```
+
+2. Install the "Linux NRPE3" Plugin Pack from "Configuration > Plugin Packs > Manager" in the WUI.
+
+<!--END_DOCUSAURUS_CODE_TABS-->
+
```

Comment:
We will keep the 'Installation' dedicated section for theses intructions. Example: https://github.com/centreon/centreon-documentation/blob/master/en/integrations/plugin-packs/procedures/network-cisco-standard-snmp.md#installation